.github/workflows/service_test_fs.yml

```diff
@@ -51,9 +51,7 @@ jobs:
       - name: Test
         shell: bash
         working-directory: core
-        run: cargo test services_fs -- --show-output
+        run: cargo test services_fs
```

Comment:
I tried it locally and it works. Why change this?

Reply:
Seems that you are missing `with: need_nextest`

Reply:
Then there is no need to change it here, right?

Reply:
By removing `show-output` we can keep the output align with `nextest`.

Reply:
We add show-output before so that we can check does this test run with credential. But after we use `github.event_name == 'push' || !github.event.pull_request.head.repo.fork`, we don't need this trick anymore.

Reply:
🤣 For such a local service,  maybe it is not needed at first.